pkg/terror/error_list.go

```diff
@@ -1057,4 +1061,7 @@ var (
 	ErrSchedulerSubTaskStageInvalidUpdate = New(codeSchedulerSubTaskStageInvalidUpdate, ClassDMMaster, ScopeInternal, LevelMedium, "invalid new expectant subtask stage %s", "")
 	ErrSchedulerSubTaskOpTaskNotExist     = New(codeSchedulerSubTaskOpTaskNotExist, ClassDMMaster, ScopeInternal, LevelMedium, "subtasks with name %s need to be operate not exist", "")
 	ErrSchedulerSubTaskOpSourceNotExist   = New(codeSchedulerSubTaskOpSourceNotExist, ClassDMMaster, ScopeInternal, LevelMedium, "sources %v need to be operate not exist", "")
+
+	// default error code
+	ErrNotSet = New(codeNotSet, ClassNotSet, ScopeNotSet, LevelHigh, "", "")
```

Comment:
where does this error happen?

Reply:
When we throw a error but not terror [here](https://github.com/pingcap/dm/blob/6c37e3da11cbd62fc48c66e55ec69f8f85dd463e/dm/unit/unit.go#L78)